include/ck_tile/ops/reduce/pipeline/reduce2d_problem.hpp

```diff
@@ -20,7 +23,11 @@ struct Reduce2dProblem
     using YDataType       = remove_cvref_t<YDataType_>;
     using BlockShape      = remove_cvref_t<BlockShape_>;
     using ReduceOp        = ReduceOp_;
+    using KeptDim         = remove_cvref_t<KeptDim_>;
```

Comment:
A general CK nit: By convention, if we have to remove_cvref_t on our class template parameters all the time, we're probably doing something wrong. We are introducing more types for the compiler when these should be handled up by the instantiation.
I'd personally like to stop using these type redefinitions in each class moving forward